students/latentgenius/README.md

```diff
@@ -0,0 +1,19 @@
+# How to use this program
```

Comment:
I'm not sure what is happening here (and the rest of the codes). You probably shouldn't update other students' codes.